src/consensus/consensus.h

```diff
@@ -28,4 +30,8 @@ enum {
     LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
 };
 
+static const int64_t SECONDS_PER_YEAR = 31558060;
```

Comment:
Orbital period of the Earth == mean solar year != mean tropical year; 1 Earth year = 365.242199 days, or 365.2425 days in Gregorian calendar